sxr-multilight/app/src/main/java/com/samsungxr/multilight/MultiLightMain.java

```diff
@@ -49,6 +51,11 @@ public void onInit(SXRContext sxrContext) {
 
         SXRNode root = new SXRNode(sxrContext);
         SXRNode character = createCharacter(sxrContext);
+        character.getTransform().setScale(0.008f,0.008f,0.008f);
```

Comment:
Why are now the scale and the translation necessary?

Reply:
@liaxim Could you test it now. It is working.